lib/Auth/Process/COmanageDbClient.php

```diff
@@ -324,62 +336,64 @@ public function process(&$state)
             }
             unset($state['Attributes']['uid']);
             $orgId = $state['Attributes'][$this->userIdAttribute][0];
-            SimpleSAML_Logger::debug("[attrauthcomanage] process: orgId=". var_export($orgId, true));
+            Logger::debug("[attrauthcomanage] process: orgId=". var_export($orgId, true));
             $basicInfo = $this->_getBasicInfo($orgId);
-            SimpleSAML_Logger::debug("[attrauthcomanage] process: basicInfo=". var_export($basicInfo, true));
-            if (empty($basicInfo['id']) || empty($basicInfo['status']) 
+            Logger::debug("[attrauthcomanage] process: basicInfo=". var_export($basicInfo, true));
+            if (empty($basicInfo['id']) || empty($basicInfo['status'])
                 || ($basicInfo['status'] !== 'A' && $basicInfo['status'] !== 'GP')) {
                   $state['UserID'] = $orgId;
-                  $params = array();
-                  $id = SimpleSAML_Auth_State::saveState($state, 'attrauthcomanage:register');
-                  $callback = SimpleSAML_Module::getModuleURL('attrauthcomanage/idp_callback.php', array('stateId' => $id));
-                  SimpleSAML_Logger::debug("[attrauthcomanage] process: callback url => " . $callback);
-                  $params = array("targetnew" => $callback);
+                  $params = [];
+                  $id = State::saveState($state, 'attrauthcomanage:register');
+                  $callback = Module::getModuleURL('attrauthcomanage/idp_callback.php', ['stateId' => $id]);
+                  Logger::debug("[attrauthcomanage] process: callback url => " . $callback);
+                  $params = [
+                    "targetnew" => $callback
+                  ];
                   if (!empty($state['saml:AuthenticatingAuthority']) && in_array(end($state['saml:AuthenticatingAuthority']), $this->communityIdps, true)) {
-                    \SimpleSAML\Utils\HTTP::redirectTrustedURL($this->registryUrls['community_sign_up'], $params);
+                      HTTP::redirectTrustedURL($this->registryUrls['community_sign_up'], $params);
                   }
                   $this->_redirect($basicInfo, $state, $params);
             }
             $loginId = $this->_getLoginId($basicInfo['id']);
-            SimpleSAML_Logger::debug("[attrauthcomanage] process: loginId=". var_export($loginId, true));
+            Logger::debug("[attrauthcomanage] process: loginId=". var_export($loginId, true));
             if ($loginId === null) {
                 // Normally, this should not happen
-                throw new Exception('There is a problem with your account. Please contact support for further assistance.');
+                throw new Error\Exception('There is a problem with your account. Please contact support for further assistance.');
             }
-            $state['Attributes'][$this->userIdAttribute] = array($loginId);
+            $state['Attributes'][$this->userIdAttribute][] = $loginId;
```

Comment:
`$state['Attributes'][$this->userIdAttribute] = [$loginId];`

Reply:
done